hw3.1/style.css

```diff
@@ -3,7 +3,7 @@
     margin-left: auto;
     margin-right: auto;
 }
-header{
+.up{
     background-color: rgb(154, 9, 9);
     height: 50px;

```

Comment:
It is better to avoid height hardcode (elements height should be as its parts inside)